prompt-service/src/unstract/prompt_service/main.py

```diff
@@ -315,102 +312,64 @@ def prompt_processor() -> Any:
                 "Fetched context from vector DB",
             )
 
-        assertion_failed = False
-        answer = "yes"
-        _publish_log(
-            log_events_id,
-            {"tool_id": tool_id, "prompt_key": prompt_name, "doc_name": doc_name},
-            LogLevel.DEBUG,
-            RunLevel.RUN,
-            "Verifying assertion prompt",
-        )
-
-        is_assert = output[PSKeys.IS_ASSERT]
-        if is_assert:
-            app.logger.debug(f'Asserting prompt: {output["assert_prompt"]}')
+        if chunk_size == 0:
```

Comment:
@chandrasekharan-zipstack Instead of removing the `assertion` logic can we just comment it out?
I understand it is done for cleaner code, but if we decide to bring back the feature it might avoid rework.

Reply:
@harini-venkataraman we've thrown away code like this previously. I suggest making note of the PR or commit instead and removing code. It becomes hard to read otherwise

Reply:
Made note of it [here](https://zipstack.atlassian.net/wiki/spaces/PT/pages/356515842/Code+Bookmarks#Removed-assertion-logic-from-Prompt-Studio-(09%2F05%2F2024))